observability/opentelemetry-api/src/main/java/org/wildfly/extension/opentelemetry/api/WildFlyOpenTelemetryConfig.java

```diff
@@ -10,8 +10,13 @@
 import java.util.Map;
 
 import io.smallrye.opentelemetry.api.OpenTelemetryConfig;
+import org.wildfly.service.descriptor.NullaryServiceDescriptor;
 
 public final class WildFlyOpenTelemetryConfig implements OpenTelemetryConfig {
+    public static NullaryServiceDescriptor<WildFlyOpenTelemetryConfig> SERVICE_DESCRIPTOR =
+            NullaryServiceDescriptor.of("org.wildfly.extension.opentelemetry.config",
+                WildFlyOpenTelemetryConfig.class);
```

Comment:
Do consumers need this specific implementation?  If not, just expose the interface.